l1-contracts/src/core/Rollup.sol

```diff
@@ -484,9 +510,25 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
     blocks[blockNumber] = BlockLog({
       archive: _archive,
       blockHash: _blockHash,
+      blobHash: blobHash,
```

Comment:
First, why the blobHash is in here? It don't seem to ever be read from here?
Second see later comment.